### PR TITLE
Add useful content to denied page

### PR DESCRIPTION
**Pull request by gracieflores**
This should help Hannah with a small issue, by providing an additional step for people to try before asking Hannah why the invite link doesn't work.

### CODE REVIEW

**Comment by gracieflores** [comment]
@ldko @somexpert This is ready for review.

File: invite/templates/invite/denied.html [hunk 1]
```diff
@@ -3,5 +3,5 @@
 Access Denied
 {% endblock %}
 {% block content %}
-That is an invalid or expired activation code.
+That is an invalid or expired activation code. You may have already registered an account. Try using the "Forgot Password" link on the edit login page to retrieve your credentials via email.
```
Comment (somexpert) [inline invite/templates/invite/denied.html:6]: Should we capitalize "edit"? Or maybe remove it entirely? I think some people don't really see it as it's own thing, but rather as a part of Aubrey.

Comment (ldko) [inline invite/templates/invite/denied.html:6]: We could change the last sentence to something like "Try retrieving your credentials via email." with part or all of that sentence linked to the amnesia page.

**Review by somexpert — APPROVED** [review]
Looks good to me! Just have the one comment, but regardless I think this is fine to merge in :thumbsup:

File: invite/templates/invite/denied.html (hunk 1 above)
Comment (ldko) [reply invite/templates/invite/denied.html:6]: Also, theoretically this app isn't a part of Edit, so to keep it more widely applicable, not referencing the Edit login page at all would be ideal.

**Review by ldko — APPROVED** [review]
This works for me, and I think it looks good :+1: . We can see what @htarver thinks before merging it.

**Comment by ldko** [comment]
@htarver does this look ok to you:
image

**Comment by htarver** [comment]
I think it looks ok, but I left out the URL on purpose -- I think it could be helpful, but not everyone is logging in to the same edit interface.

**Comment by ldko** [comment]
The URL is not hard-coded, so I don't think it should matter which edit interface they are using.

**Pull request merged**